changelog.md

```diff
@@ -1,25 +1,60 @@
-# 4.0.6 / 2022-01-18
+# 4.1.0 / 2022-01-21
+
+### New Features
+
+- New kanban feature
+- Improvement of the responsive aspect of the interface
+``
+(responsive is an adjective)
+- News feed reversed
+- Calibration of the cover and avatar profile photos
+- Improvement of user's space management for the admin
+- Improvement of the call feature (bugfix and polish)
+- UI/UX improvement (buttons, redirections)
+- Performance improvement: better image and PDF preview generation, loading of space list and user list
+- Reactivation of the collabora draw functionality
```

Comment:
Sort new feature by more important, better on top and less important at the end

Reply:
What do you think about this order ?

- New kanban feature
- Calibration of the cover and avatar profile photos
- Improvement of user's space management for the admin
- News feed reversed
- Reactivation of the collabora draw functionality
- Improvement of the responsive aspect of the interface
- Improvement of the call feature (bugfix and polish)
- UI/UX improvement (buttons, redirections)
- Performance improvement: better image and PDF preview generation, loading of space list and user list

Reply:
I'd have swap pur ``Reactivation of the collabora draw functionality`` in third position
Like features first, UI second, performances third

Reply:
- New kanban feature
- Improvement of the responsive aspect of the interface
- News feed now displays newest news first
- Crop feature for the cover and avatar profile photos
- Improvement of user's space management for the admin
- Improvement of the call feature (bugfix and polish)
- UI/UX improvement (buttons, redirections)
- Performance improvements:
  - better image and PDF preview generation
  - loading speed of spaces and users lists
- Reactivation of the draw functionality in the collaborative document

Reply:
Validated with the rest of the team